pkg/blockatlas/lending.go

```diff
@@ -0,0 +1,59 @@
+package blockatlas
+
+type ProviderType string
+
+const (
+	ProviderTypeLending ProviderType = "lending"
```

Comment:
remove Type in namings of objects

Reply:
Unfortunately this is in common 'blockatlas' package (bad structure), and I don't want to pollute that with short 'Lending'.